package.json

```diff
@@ -6,11 +6,10 @@
   "main": "src/Frontend/App.js",
   "scripts": {
     "dev": "NODE_ENV=development npm run frontend",
-    "frontend": "webpack",
+    "frontend": "webpack --watch",
```

Comment:
this is not a good idea as the `prod` command relies on this command and the `prod` command is used during the deploy (https://github.com/Sententiaregum/Sententiaregum/blob/master/src/AppBundle/Composer/ScriptHandler.php#L57) and no watcher should be used when deploying with composer.